glide.lock

```diff
@@ -49,15 +50,26 @@ imports:
 - name: github.com/coreos/yaml
   version: 6b16a5714269b2f70720a45406b1babd947a17ef
 - name: github.com/go-ini/ini
-  version: 6e4869b434bd001f6983749881c7ead3545887d8
+  version: 2e44421e256d82ebbf3d4d4fcabe8930b905eff3
 - name: github.com/inconshreveable/mousetrap
   version: 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
 - name: github.com/jmespath/go-jmespath
-  version: bd40a432e4c76585ef6b72d3fd96fb9b6dc7b68d
+  version: 3433f3ea46d9f8019119e7dd41274e112a2359a9
+- name: github.com/Masterminds/semver
+  version: 59c29afe1a994eacb71c833025ca7acf874bb1da
+- name: github.com/Masterminds/sprig
+  version: 23597e5f6ad0e4d590e71314bfd0251a4a3cf849
+- name: github.com/satori/go.uuid
+  version: 879c5887cd475cd7864858769793b2ceb0d44feb
 - name: github.com/spf13/cobra
   version: 856b96dcb49d6427babe192998a35190a12c2230
 - name: github.com/spf13/pflag
   version: dabebe21bf790f782ea4c7bbd2efc430de182afd
+- name: golang.org/x/crypto
+  version: 1f22c0103821b9390939b6776727195525381532
+  subpackages:
+  - pbkdf2
+  - scrypt
 - name: gopkg.in/yaml.v2
-  version: a5b47d31c556af34a302ce5d659e6fea44d90de0
+  version: a83829b6f1293c91addabc89d0571c246397bbf4
```

Comment:
Ah! If I remember correctly, although the gopkg.in is just a proxy for the repo in GitHub, the go-yaml v2 package served by gopkg.in has been a bit out-dated.
I believe I had manually modified the `version` after I ran `glide update` last time.

I guess it will be safe to rewrite the version manually back to `a5b47d31c556af34a302ce5d659e6fea44d90de0` after you run `glide get`.